app/lib/pdf_filler/nj1040_pdf.rb

```diff
@@ -0,0 +1,111 @@
+module PdfFiller
+  class Nj1040Pdf
+    include PdfHelper
+
+    def source_pdf_name
+      "nj1040-TY2023"
+    end
+
+    def initialize(submission)
+      @submission = submission
+
+      # Most PDF fields are grabbed right off the XML
+      builder = StateFile::StateInformationService.submission_builder_class(:nj)
+      @xml_document = builder.new(submission).document
+    end
+
+    def hash_for_pdf
+      county_code = get_county_code
+      taxpayer_ssn = get_taxpayer_ssn
+      spouse_ssn = get_spouse_ssn
+      {
+        # county code
+        "CM4": county_code[1],
+        "CM3": county_code[2],
+        "CM2": county_code[3],
+        "CM1": county_code[4],
+
+        # taxpayer ssn
+        "undefined": taxpayer_ssn[0],
```

Comment:
[DUST] How can we verify that these field names map to the correct fields in the PDF? The `pdf fields.txt` document in the Google drive shows the output of `pdftk 1040.pdf dump_data_fields` but it isn't very helpful for determining which fields each name maps to.